ac_dc/tests/test_govt_anonymization.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+import anonymization
+
+
+@pytest.mark.parametrize(
+    "sentence,lang_id,expected",
+    [
+        ("hello world", "en", "hello world"),
+        ("my government id is 123 123 123", "en", "my government id is 111 111 111"),
```

Comment:
@ontocord can you help add unit tests here :)? at the same time we can sanity check that the regexes are doing what we're expecting

Reply:
@paulovn i noticed you had very similar tests in your pii-manager. we could potentially combine both eventually i think

Reply:
@paulovn i noticed some similar tests in your `pii-manager` PR, i think we can eventually combine both :)

Reply:
yes, that's the actual idea -- consolidate my package, migrate the current GOVT_ID detectors there, and then add new ones as we develop them. The structure of the package is designed to let people do small PRs with a single PII or a couple of them in parallel

Reply:
You can use the faker library to generate fake govt id. I'm not going to spend too much time on checking checksums and things like that since the main goal is to capture what *could* be govt_ids b.c they are risky. For govt id at least, i think we can go for more recall and less accuracy. But since we aren't evaling on these types of numbers, being overborad, in my opinion will not hurt downstream performance. lmk if you disagree. This is different than a generic PII library which should have high accuracy.

Reply:
For higher accuracy we can use the govtid library @paulovn found as a secondary test. But again, I think that would be for a different type of application than here, which is basic dataset filtering.

Reply:
i'm okay with that